core/examples/json/crawlable_catalogue/collection.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "example-collection",
+    "title": "Dataset and Imagery Record examples",
+    "description": "This Collection links down to several examples of crawlable record collections of different types",
+    "extent":
+    {
+        "spatial":
+        {
+            "bbox":
+            [
+                [
+                    -180,
+                    -90,
+                    180,
+                    90
```

Comment:
This should be the minimal extent that includes all records. Or will the imagery collection cover the whole world?

Reply:
The imagery one is marked as the whole world, so since this was a super class it seemed like it should include it all. I agree a better example would be a limited imagery catalog with valid bounds.